app/javascript/chat/chat.jsx

```diff
@@ -1634,6 +1646,10 @@ export default class Chat extends Component {
     const { activeChannel } = this.state;
     const mention = e.keyCode === 64;
     if (mention && activeChannel.channel_type !== 'direct') {
+      const memberListElemt = document.getElementById('mentionList');
```

Comment:
nitpick: Name this `memberList` or `memberListElement`. There is no need for the abbreviated variable name as it get minified when bundled.